main.rb

```diff
@@ -31,6 +22,50 @@ def list_people
     puts
   end
 
+  def list_rentals_for_person
+    print 'ID of person: '
+    id_of_person = gets.chomp.to_i
+    puts 'Rentals:'
+    @rentals.each do |rent|
+      puts "Date: #{rent.date}, Book #{rent.book.title} by #{rent.book.author}" if rent.person.id == id_of_person
+    end
+    puts
+  end
+end
+
+class CreateClass
+  def initialize(list_handler)
+    @list_handler = list_handler
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    b = Book.new(title, author)
+    @list_handler.books.push(b)
+    puts 'Book created successfully'
+    puts
+  end
```

Comment:
- [ ] the purpose of this assignment is to make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit). what is meant by this is that you are expected to create separate files for the classes handling the logic. What this might look like is a separate file handling all the book methods, one for the person methods and one for the rental methods.